worlds/lufia2ac/test/TestCustomItemPool.py

```diff
@@ -53,5 +53,5 @@ def test_verify_item_name(self) -> None:
                                "Item The car blade from option CustomItemPool\\(The car blade: 2\\) is not a "
                                "valid item name from Lufia II Ancient Cave\\. Did you mean 'Dekar blade'",
                                lambda: handle_option(Namespace(game="Lufia II Ancient Cave", name="Player"),
-                                                     self.options, "custom_item_pool", CustomItemPool,
+                                                     self.options, "custom_item_pool", {"custom_item_pool": CustomItemPool},
```

Comment:
I'm not convinced that this breaking change is necessary. (It might break other unsupported apworlds too.)

If creating a new `dict` is ok here, why couldn't `handle_option` make the new `dict` inside?

The only `ChoiceRecord` method that uses it only references 1 specific `Option` from a parameter. Couldn't that `Option` be passed to that method as that parameter?

I don't think any change to lufia2ac or any other world should be needed.